docs/faq/precision.rst

```diff
@@ -0,0 +1,44 @@
+.. doctest-skip-all
+
+.. _precision:
+
+*********
+Precision
+*********
+
+Rise/set/meridian transit calculations in `astroplan` are designed to be fast
+while achieving a precision comparable to what can be predicted given the
+affects of the changing atmosphere. As a result, there may be some slightly
+counter-intuitive behavior in `astroplan` methods like
+`astroplan.Observer.target_rise_time`, `astroplan.Observer.target_set_time` and
+`astroplan.Observer.target_meridian_transit_time`.
+
+For example, to calculate the rise time of Sirius, you might write::
+
+    from astroplan import Observer, FixedTarget
+    from astropy.time import Time
+
+    # Set up observer, target, and time
+    keck = Observer.at_site("Keck")
+    sirius = FixedTarget.from_name("Sirius")
+    time = Time('2010-05-11 06:00:00')
+
+    # Find rise time of Sirius at Keck nearest to `time`
+    rise_time = keck.target_rise_time(time, sirius)
+
+You might expect the altitude of Sirius to be zero degrees at ``rise_time``,
+i.e. Sirius will be on the horizon, but this is not the case::
+
+    >>> altitude_at_rise = keck.altaz(rise_time, sirius).alt
+    >>> print(altitude_at_rise.to('arcsec'))
+    2.70185arcsec
+
+The rise and set time methods use the following approximation:
+
+* A time series of altitudes for the target is computed at times near ``time``
+
+* The two times when the target is nearest to the horizon are identified, and a
+  linear interpolation is done between those times to find the horizon-crossing
+
+This method has a precision of roughly a few arcseconds, so your targets may
+be slightly above or below the horizon at their rise or set times.
```

Comment:
could also just link to the IERS doc (I think ``:doc:`iers` `` will do the trick, although it may need to be ``:doc:`/faq/iers` `` or something like that)